src/REPLMode/command_declarations.jl

```diff
@@ -311,14 +311,14 @@ packages will not be upgraded at all.
 """,
 ],
 PSA[:name => "generate",
-    :api => API.generate_deprecated,
+    :api => API.generate,
     :arg_count => 1 => 1,
     :arg_parser => ((x,y) -> map(expanduser, unwrap(x))),
     :description => "generate files for a new project",
     :help => md"""
     generate pkgname
 
-Create a project called `pkgname` in the current folder.
+Create a minimal project called `pkgname` in the current folder. For more featureful package creation, please see `PkgTemplates.jl`.
```

Comment:
I prefer to not ship this redirection in the docstring, but instead in the documentation. We have no idea whether it is still `PkgTemplates.jl` five years later.

Reply:
Sure, but keep in mind that currently, `] generate` is deprecated in favor of PkgTemplates.jl with the deprecation warning: `Pkg.generate is deprecated. Please use PkgTemplates.jl instead.`

Incidentally, I think that makes a good case for having this "core" functionality built-in to Pkg.jl.